### PR TITLE
1.0: Add linux capability descriptions

### PR DESCRIPTION
**Pull request by cosmo0920**
Related to https://github.com/fluent/fluentd/pull/3155

### CODE REVIEW

**Comment by cosmo0920** [comment]
@repeatedly Could you kindly take a look?
Or, unifying documentation PR with #249 is needed?

File: configuration/linux-capability.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+* Ruby and its development packages
+  * ruby-dev on Debian GNU/Linux and Ubuntu
+  * ruby-devel on CentOS 7/8, Fedora 33, AmazonLinux 2
+* `setcap` command
+  * libcap2-bin on Debian GNU/Linux and Ubuntu
+  * libcap on CentOS 7/8, Fedora 33, AmazonLinux 2
+* `filecap` command
+  * libcap-ng-utils on Debian GNU/Linux and Ubuntu
+  * libcap-ng-utils on CentOS 7/8, Fedora 33, AmazonLinux 2
+
```
Comment (repeatedly) [inline configuration/linux-capability.md:21]: Should mention `Fluentd v1.12 or later`.

Comment (cosmo0920) [reply configuration/linux-capability.md:21]: Oops. I'll add it.

Comment (cosmo0920) [reply configuration/linux-capability.md:22]: Fixed https://github.com/fluent/fluentd-docs-gitbook/pull/248/commits/16f9e0e93bb60b54b352769656a1524177bb215b

File: configuration/linux-capability.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+$ sudo mkdir /var/run/fluentd
+$ sudo chown `whoami` /var/run/fluentd
+```
+
+Then, run as ordinal user with `cap_dac_read_search` capability attached Ruby:
+
+```console
+$ bundle exec fluentd -c in_tail_camouflage_permission.conf
+2020-11-05 14:47:57 +0900 [info]: parsing config file is succeeded path="example/in_tail.conf"
+2020-11-05 14:47:57 +0900 [info]: gem 'fluentd' version '1.11.4'
```
Comment (repeatedly) [inline configuration/linux-capability.md:141]: version should be `1.12.0`.

File: configuration/linux-capability.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+    <parse>
+      @type "syslog"
+      unmatched_lines
+    </parse>
+  </source>
+  <match test>
+    @type stdout
+  </match>
+</ROOT>
+2020-11-05 14:47:57 +0900 [info]: starting fluentd-1.11.4 pid=12109 ruby="2.6.3"
```
Comment (repeatedly) [inline configuration/linux-capability.md:161]: ditto

File: configuration/linux-capability.md (hunk 2 above)
Comment (cosmo0920) [reply configuration/linux-capability.md]: Oh! :fearful:

Comment (cosmo0920) [reply configuration/linux-capability.md]: Fixed https://github.com/fluent/fluentd-docs-gitbook/pull/248/commits/3441bbe75cc492ab77eb87e0143b8d65ac099f7e

**Review by repeatedly — APPROVED** [review]

**Pull request merged**